tests/test_image_transforms.py

```diff
@@ -313,6 +322,46 @@ def test_normalize(self):
             np.allclose(normalize(image, mean=mean, std=std, input_data_format="channels_last"), expected_image)
         )
 
+    @parameterized.expand(
+        [
+            ((3, 224, 224), (120, 60)),
+            ((3, 223, 223), (120, 60)),
+            ((3, 224, 224), (121, 61)),
+            ((3, 223, 223), (121, 61)),
+        ]
+    )
+    @require_torchvision
+    def test_center_crop_with_torchvision(self, image_size, expected_size):
+        # Define input image size
+        image_channels = image_size[0]
+        # Define expected output size
+        expected_height = expected_size[0]
+        expected_width = expected_size[1]
+        expected_channels = image_channels
+
+        # Defining input image
+        image = np.random.randint(0, 256, image_size, np.uint8)
+        pil_image = PIL.Image.fromarray(np.transpose(image, (1, 2, 0)))
+
+        # Test that exception is raised if inputs are incorrect
+        with self.assertRaises(ValueError):
+            center_crop(image, 10)
+
+        # Use torchvision CenterCrop transformation as reference
+        transf_center_crop = transforms.CenterCrop(expected_size)
+
+        # Apply torchvision CenterCrop transformation
+        expected_output = transf_center_crop(pil_image)
+        expected_output_np = np.array(expected_output)
+
+        # Apply center_crop tranformation
+        cropped_image = center_crop(image, expected_size, data_format="channels_last")
+
+        # Test that output is correct and matches torchvision
+        self.assertIsInstance(cropped_image, np.ndarray)
+        self.assertEqual(cropped_image.shape, (expected_height, expected_width, expected_channels))
+        self.assertTrue(np.allclose(cropped_image, expected_output_np))
```

Comment:
Cool! Can you also add  raw numbers to have a better idea of what we are actually expecting here! 😉

Reply:
If I understood correctly, you're suggesting a hard-coded numpy image, so we can explicitly see what we expect to have, right?

If so, yes, I can do that. But I think it would be better to have it in a new test function, as this one creates the numpy image with random values.

Please, let me know if that's what you're expecting, and I will do it. :)

Reply:
Haha right right, i mean we can hardcode them if they are not too long (like long tensor slices) but otherwise no worries!

Reply:
Thanks. I got the idea.
I will make it with slices :)